packages/zipkin-instrumentation-memcached/src/zipkinClient.js

```diff
@@ -17,6 +17,7 @@ module.exports = function zipkinClient(
         }));
         tracer.recordAnnotation(new Annotation.ClientRecv());
       });
+      tracer.setId(originalID);
```

Comment:
so this line implies a leak I guess

Reply:
Correct

Reply:
I shared small service on [issue 172](https://github.com/openzipkin/zipkin-js/issues/176) to prove my point

Reply:
visual check, all calls to setId happen in scoped blocks, so this guard is indeed interesting

Reply:
the test passes with this commented out.. however the test uses explicit context, not CLS. CLS is the scenario of your example I think...

Reply:
this also passes with CLS..